distributed/dashboard/scheduler.py

```diff
@@ -269,7 +271,7 @@ def collect(self):
         yield GaugeMetricFamily(
             "dask_scheduler_desired_workers",
             "Number of workers scheduler needs for task graph.",
-            value=self.server.adaptive_target(),
+            value=self.server.adaptive_target(adaptive_target_duration),
```

Comment:
Rather than get the configuration value every time this method is used, lets get the configuration value within the function itself.  That is likely to affect things more consistently, and keep things a bit more maintainable so that we don't need to always remember to do this logic each time we call this function.